README.md

```diff
@@ -1,27 +1,29 @@
 
-# kured - Kubernetes Reboot Daemon
+# kured - Kubernetes Reboot Daemon  <!-- omit in toc -->
```

Comment:
Not sure we need this, and if we do, it should be a separate commit explaining why we do this.